prepare/templates/qa/with_context.py

```diff
@@ -63,6 +64,32 @@
     overwrite=True,
 )
 
+add_to_catalog(
+    MultiReferenceTemplate(
+        instruction="Answer the question based on the information provided in the {context_type} given below.",
+        input_format="{context_type}:\n{context}\nQuestion:\n{question}",
+        output_format="{answer}",
+        target_prefix="Answer:\n",
+        references_field="answers",
+        title_fields=["context_type"],
+    ),
+    "templates.qa.with_context",
+    overwrite=True,
+)
+
+add_to_catalog(
+    MultiReferenceTemplate(
+        instruction="Answer the question directly based on the information provided in the {context_type}. Respond with the exact phrase within the {context_type} that provides the answer.",
```

Comment:
Maybe:
Extract the exact phrase from the {context_type} that directly answers the question, without any alterations.

Reply:
To make it more clear..